site/docs/install-compile-source.md

```diff
@@ -140,7 +141,7 @@ Unix-like systems.)
 4.  The output will be at `bazel-bin\src\bazel-dev.exe` (or
     `bazel-bin\src\bazel.exe`).
 
-<h3 id="build-bazel-install">5. Install the built binary</h3>
+<h3 id="build-bazel-install">Step 5: Install the built binary</h3>
 
 Actually, there's nothing to install.
```

Comment:
I disagree with this.  You can't keep using the bazel in bazel-bin. You need to 

  You should 
- copy the built binary to some place in your path, making sure it will be selected instead of the previously downloaded bazel
- to bazel test  ???? on the bazel source tree. Make sure all tests pass
Continue to use that new bazel.

Reply:
Points taken. I'd like to restrict this PR to formatting, and save deeper content refactoring / splitting up pages for the next phase's effort.